directories/src/utils/app.js

```diff
@@ -0,0 +1,5 @@
+const randomNumber = (maxNumber) => {
+  return Math.floor(Math.random() * (maxNumber + 1));
```

Comment:
Same here about `Math.random()` not being secure.

Reply:
the same